components/form-toggle/style.scss

```diff
@@ -32,25 +32,34 @@ $toggle-border-width: 2px;
 		}
 	}
 
-	&:after {
-		content: '';
+	&__thumb {
```

Comment:
This seems a little bit obtuse, coding style wise. I admit to not being 100% sure about exactly what the coding guidelines would say, but I have a feeling it would be better to replace `&__thumb` with just plainly `.components-form-toggle__thumb`.